src/main/java/frc/robot/subsystems/shooter/ShooterConstants.java

```diff
@@ -0,0 +1,5 @@
+package frc.robot.subsystems.shooter;
+
+public class ShooterConstants {
+    public static double atSetpointTolerance = 0.05; // Percentages
```

Comment:
This is probably too high, a 5% tolerance means that with a setpoint of 1000RPM, we'll be fine with 1050RPM which might be a significant error. We will probably achieve 0.5% tolerance.

Reply:
Skibidi yes yes

Reply:
Also, I'm not sure the error should be percentage but rather in rps. The farther we are the more consistent the velocity would need to be, hence the error would need to be smaller. Using percentages means the opposite.

Reply:
That's how `isNear()` works though. I think we can see in testing whether the tolerance is proportional or constant. That means, if the setpoint is 100 then 101 might be fine, and if the setpoint is 1000 then 1010 might be fine too.

Reply:
Ok, we'll leave it for now.